components/StationStatus.js

```diff
@@ -84,7 +84,13 @@ const styles = StyleSheet.create({
     fontSize: 18,
     color: COLOR_PRIMARY,
     fontWeight: 'bold',
+    opacity: .7,
+    fontStyle: 'italic',
   },
+  updateText: {
+    opacity: .7,
+    color: COLOR_PRIMARY,
+  }
```

Comment:
Missing trailing comma.

[Read more about it here.](http://eslint.org/docs/rules/comma-dangle)